packages/manager/apps/web/client/app/components/manager-order-catalog-price/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+import component from './manager-order-catalog-price.component';
+
+const moduleName = 'ovhManagerOrderCatalogPrice';
+
+angular
+  .module(moduleName, [])
+  .component('ovhManagerOrderCatalogPrice', component)
```

Comment:
I think this could be extracted to `web-universe-components` at least, that way it could be used for other orders elesewhere